packages/css/src/components/page/page.scss

```
@@ -0,0 +1,12 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+.ams-page {
+  background-color: var(--ams-page-background-color);
+  margin-inline: auto;
+  max-inline-size: var(--ams-page-max-inline-size);
+  min-block-size: 100dvh;
```

`dvh` is still '[newly available](https://caniuse.com/viewport-unit-variants)'. Let's add a fallback for browsers that do not support it. 

```suggestion
  min-block-size: 100vh; /* Fallback for browsers that do not support dvh */
  min-block-size: 100dvh;
```